patterns/testimonial.php

```diff
@@ -7,31 +7,31 @@
-<h3 class="has-tertiary-color has-text-color has-medium-font-size" style="line-height:1;text-transform:uppercase">What students say</h3>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"80px","bottom":"120px","right":"20px","left":"20px"},"blockGap":"0px"}},"backgroundColor":"primary","textColor":"background","layout":{"type":"constrained","contentSize":"1000px"}} -->
+<div class="wp-block-group alignfull has-background-color has-primary-background-color has-text-color has-background" style="padding-top:80px;padding-right:20px;padding-bottom:120px;padding-left:20px"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px","bottom":"0px"}},"border":{"top":{"width":"0px","style":"none"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"color":"var:preset|color|tertiary","width":"1px"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-bottom:0px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase","lineHeight":"1","letterSpacing":"0.01em"}},"fontSize":"medium","fontFamily":"league-gothic"} -->
+<h3 class="has-league-gothic-font-family has-medium-font-size" style="letter-spacing:0.01em;line-height:1;text-transform:uppercase">What students say</h3>
 <!-- /wp:heading --></div>
 <!-- /wp:group -->
 
-<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"35px","left":"35px"},"padding":{"top":"40px"}}},"textColor":"tertiary"} -->
-<div class="wp-block-columns has-tertiary-color has-text-color" style="padding-top:40px"><!-- wp:column {"verticalAlignment":"bottom","width":"41%"} -->
+<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"35px","left":"35px"},"padding":{"top":"40px"}}}} -->
+<div class="wp-block-columns" style="padding-top:40px"><!-- wp:column {"verticalAlignment":"bottom","width":"41%"} -->
 <div class="wp-block-column is-vertically-aligned-bottom" style="flex-basis:41%"><!-- wp:image {"id":2150,"sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"8px","width":"1px"}}} -->
-<figure class="wp-block-image size-full has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/testimonial.png" alt="" class="wp-image-2150" style="border-width:1px;border-radius:8px"/></figure>
+<figure class="wp-block-image size-full has-custom-border"><img src="http://senseitheme.local/wp-content/themes/course/assets/images/testimonial.png" alt="" class="wp-image-2150" style="border-width:1px;border-radius:8px"/></figure>
```

Comment:
Small issue here I think, looks like the local URL got slipped in, so the image does not load for testimonial

Reply:
The line on the left looks a bit taller here
![image](https://user-images.githubusercontent.com/6820724/201381096-85090842-b654-4794-b8bc-8b488e29d293.png)

Reply:
Fixed by https://github.com/Automattic/course/pull/86/commits/043a047db70bc3f433222f7e40ce24d63355c004 and https://github.com/Automattic/course/pull/86/commits/4c035384ff68c0e0759d39fc1b85cedadd105506.

Reply:
Perfect! Looks great now